react/features/base/tracks/actions.ts

```diff
@@ -591,11 +595,14 @@ export function trackVideoStarted(track: any) {
  * }}
  */
 export function trackVideoTypeChanged(track: any, videoType: VideoType) {
+    const m = videoType === VIDEO_TYPE.CAMERA ? MEDIA_TYPE.VIDEO : MEDIA_TYPE.SCREENSHARE;
```

Comment:
The constant can be renamed to mediaType and property value shorthand syntax can be used.

Reply:
fixed.